.distfiles

```diff
@@ -54,3 +54,4 @@
 /vendor/woocommerce/action-scheduler/**/*.php
 /node_modules/php-date-formatter/js/php-date-formatter.js
 /node_modules/php-date-formatter/js/php-date-formatter.min.js
+/node_modules/moment/dist/*.js
```

Comment:
Until the bug on pup is fixed, this will probably need to go into the .distfiles of parent plugins as well. ET and TEC.